### DISCUSSION TITLE
WebGIS6 - WMS-Dienst Hintergrundkarte - Vorschaubild

### DISCUSSION THREAD

**Discussion by Andreas-Lampert** [e-netze/webgis-community#199] [Q&A]
Hallo Jürgen!

Ich versuche einen WMS-Dienst als Hintergrundkarte zu konfigurieren. Die Karte wird problemlos dargestellt, allerdings wird das Vorschaubild nicht angezeigt (bleibt weiß). 
![image](https://github.com/user-attachments/assets/afc73a3f-0c28-48a2-8f28-97eb40ff51ba)
![image](https://github.com/user-attachments/assets/aea23b45-7bee-41b3-96c0-0ebd1109ca82)

In der Entwicklungsumgebung ist bei der URL für die Karte und der URL für das Vorschaubild ein Unterschied sichtbar. 

URL des Kartenbildes:
![image](https://github.com/user-attachments/assets/4650553f-8c2d-42f6-8abf-e5c0d25231eb)

URL des Vorschaubildes:
![image](https://github.com/user-attachments/assets/4526c2c7-f281-431c-9988-ebbe6574d52f)

In der URL vom Vorschaubild ist bei crs= kein Wert hinterlegt (...&crs=&scale=...). Damit wird auch kein Bild zurückgeliefert. Erst wenn ich die URL im Browser manuell mit einem Wert bei crs ergänze (...&crs=3857&scale=...) wird ein Bild angezeigt. 

Muss ich das CRS irgendwo im CMS noch separat konfigurieren? Zum Reproduzieren könntest du z.B. den WMS-Dienst der Austro Control verwenden --> https://sdigeo-free.austrocontrol.at/geoserver/tn-a/wms?service=WMS&version=1.3.0&request=GetCapabilities

Danke!

Lg, Andi

**Comment by gis-eni** [comment]
Mit Build 6.24.3801 kann im CMS für Hintergrunddienste die Url zu einer Bild für die Kachel angegeben werden:

Dienst >> Erweiterte Einstellungen >> Basemap

![image](https://github.com/user-attachments/assets/92c56f69-2df9-41ef-b840-c765f6e86d46)

Hinweis: Dynamische und autoatische Kacheln funktionieren nur mit Raster-Tiling Diesten. Für WMS etc muss immer ein statisches Bild angegeben werden.
